### ISSUE TITLE
unable to run storybook (probably a bigger issue than just storybook)

### ISSUE THREAD

**Issue by anakinjay** [yarnpkg/yarn#875]
**BUG:**
Current behavior:
After installing react storybook and attempting to execute "yarn run storybook" I get the following errors messages:

yarn run v0.15.1
$ "start-storybook -p 6006" 
sh: start-storybook -p 6006: command not found
error Command failed with exit code 127.
info Visit http://yarnpkg.com/en/docs/cli/run for documentation about this command.

**package.json:**

{
  "name": "yarnTest",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "babel": "^6.5.2",
    "bootstrap": "^3.3.7",
    "bulma": "^0.2.1",
    "react": "^15.3.2",
    "react-dom": "^15.3.2",
    "redux": "^3.6.0"
  },
  "devDependencies": {
    "@kadira/storybook": "^2.5.2"
  },
  "scripts": {
    "storybook": "start-storybook -p 6006",
    "build-storybook": "build-storybook"
  }
}

**Expected behavior:** 
storybook server starts

**node version:**
6.2.2

**yarn version:**
0.15.1

**Comment by rjmunro** [comment]
Might this be related to #614? Is node_modules/.bin not being set up correctly, or not added to the PATH that scripts are called with?

**Comment by anakinjay** [comment]
Yep, seems like a dupe of #735

**Comment by AlicanC** [comment]
Probably fixed in #809.

**Comment by guitarmanvt** [comment]
IMHO, #1156 is the underlying cause: yarn is not passing the options through to the shell properly. Fix that, and this issue will also be fixed.

**Comment by busches** [comment]
This was fixed in `0.16.0`

```
yarn run storybook
yarn run v0.16.1
$ start-storybook -p 6006
=> Using default webpack setup based on "Create React App".
=> Loading custom webpack config (full-control mode).

React Storybook started on => http://localhost:6006/

 We will collect some anonymous usage stats of how you use storybook.
 See why?: https://getstorybook.io/tracking

webpack built 8190f68f135080310b03 in 6529ms
```

**Comment by busches** [comment]
@kittens can this be closed? This was fixed back with `0.16.0`.
